Packs/SmartSheetITAdmin/Integrations/Smartsheet_IT_Admin/Smartsheet_IT_Admin.py

```diff
@@ -0,0 +1,516 @@
+    """
+    Returning 'ok' indicates that the integration works like it is supposed to. Connection to the service is successful.
+
+    Args:
+        client: Smartsheeet client
+
+    Returns:
+        'ok' if test passed, anything else will fail the test.
+    """
+    args
+
+    res = client.user.get_current_user()
+    res_json = json.loads(str(res))
+
+    if (res_json.get('id', None) is not None):
+        return 'ok', None, None
+    else:
+        message = res_json.get('result').get('message')
+        return "Test failed because: {}.".format(message), None, None
+
+
+def get_user_command(client, args):
+    """
+        Returning user GET details and status of response.
+
+        Args:   demisto command line argument
+        client: Smartsheet
+
+        Returns:
+            success : success=True, id, email, login as username, details, active status
+            fail : success=False, id, login as username, errorCod, errorMessage, details
+    """
+    scim = verify_and_load_scim_data(args.get('scim'))
+    scim_flat_data = map_scim(scim)
+    user_id = scim_flat_data.get('id')
+    email = scim_flat_data.get('email')
+
+    if not user_id and not email:
+        raise Exception('You must provide either the id or email of the user')
+
+    if user_id:
+        res = client.get_user(user_id=user_id)
+        res_json = json.loads(str(res))
+    else:
+        res = client.get_user(email=email)
+        res_json = json.loads(str(res))
+        res_json = res_json.get('data')
+        if res_json:
+            res_json = res_json[0]
+
+    if not res_json:
+        generic_iam_context = OutputContext(success=False, iden=user_id, email=email, errorCode=404,
+                                            errorMessage=USER_NOT_FOUND, details=res_json)
+    elif res_json.get('id', None) is not None:
+        id = res_json.get('id')
+        email = res_json.get('email')
+        active = res_json.get('status', False)
+        generic_iam_context = OutputContext(success=True, iden=id, email=email, details=res_json,
+                                            active=active)
+    elif res_json.get('result').get('statusCode') == 404:
+        generic_iam_context = OutputContext(success=False, iden=user_id, errorCode=404,
+                                            errorMessage=USER_NOT_FOUND, details=res_json)
+    else:
+        errorMessage = res_json.get('result').get('message')
+        errorCode = res_json.get('result').get('statusCode')
+        generic_iam_context = OutputContext(success=False, iden=user_id, errorCode=errorCode,
+                                            errorMessage=errorMessage, details=res_json)
+
+    generic_iam_context_dt = "{}(val.id == obj.id && val.instanceName == obj.instanceName)".format(generic_iam_context.command)
+
+    outputs = {
+        generic_iam_context_dt: generic_iam_context.data
+    }
+
+    readable_output = tableToMarkdown('Get Smartsheet User:', generic_iam_context.data)
```

Comment:
maybe a better title would be something like "Smartsheet user {id} data". you might also use the tableToMarkdown attributes we talked about (headers, removeNull)

Reply:
Updated